.gitignore

```diff
@@ -1 +1,2 @@
 coverage.txt
+.idea/
```

Comment:
```suggestion
```

Reply:
even better, this should go in a personal .gitignore_global and not pollute the public one.